public/app/plugins/datasource/prometheus/datasource.tsx

```diff
@@ -1016,11 +1017,20 @@ export class PrometheusDatasource
 
   async areExemplarsAvailable() {
     try {
-      const res = await this.getResource('/api/v1/query_exemplars', {
+      const url = '/api/v1/query_exemplars';
+      const params = {
         query: 'test',
-        start: dateTime().subtract(30, 'minutes').valueOf(),
-        end: dateTime().valueOf(),
-      });
+        start: dateTime().subtract(30, 'minutes').valueOf().toString(),
+        end: dateTime().valueOf().toString(),
+      };
+      const res = await lastValueFrom(
+        this._request<PromDataSuccessResponse<PromExemplarData>>(url, params, {
+          method: 'GET',
+          // Avoid alerting the user if this test fails
+          hideFromInspector: true,
+          showErrorAlert: false,
+        })
+      );
```

Comment:
This use of this._request helps us to suppress the error associated with:
/api/v1/query_exemplars

Reply:
@leventebalogh how close is this PR https://github.com/grafana/grafana/pull/51316? Would probably help us here.

Reply:
> This use of this._request helps us to suppress the error associated with: `/api/v1/query_exemplars`

FWIW, the `/api/v1/query_exemplars` endpoint in Mimir and GEM supports multiple tenants if enabled by configuration (since 2.0).